packages/paste-core/components/form/src/select/Option.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import css from '@styled-system/css';
+import {safelySpreadFormControlProps, PROPS_TO_BLOCK} from '../shared/Utils';
+
+export interface OptionProps extends React.OptionHTMLAttributes<HTMLOptionElement> {
+  value: string;
+  children: React.ReactNode;
+}
+
+const OPTION_PROPS_TO_BLOCK = [...PROPS_TO_BLOCK, 'selected'];
+
+const OptionElement = styled.option(
+  css({
+    background: 'transparent',
+    color: 'colorText',
+  })
+);
```

Comment:
Was running this locally and gave the options a little more space to breath with 

```
const OptionElement = styled.option(
  css({
    background: 'transparent',
    color: 'colorText',
    fontFamily: 'fontFamilyText',
    lineHeight: 'lineHeight50',
    paddingTop: 'space30',
    paddingBottom: 'space30',
    paddingLeft: 'space20',
  })
);
```
![image](https://user-images.githubusercontent.com/368249/80732886-cc5a1b80-8ac1-11ea-806c-a71b51ac7c68.png)